stdlib/filepath/src/main/java/org/ballerinalang/stdlib/filepath/Utils.java

```diff
@@ -35,41 +31,24 @@
 
     static final String UNKNOWN_MESSAGE = "Unknown Error";
 
-    /**
-     * Returns error record for input reason. Error type is generic ballerina error type. This utility to construct
-     * error struct from message.
-     *
-     * @param reason Reason for creating the error object. If the reason is null, "UNKNOWN" sets by default.
-     * @param error  Java throwable object to capture description of error struct. If throwable object is null, "Unknown
-     *               Error" sets to message by default.
-     * @return Ballerina error object.
-     */
-    public static ErrorValue getPathError(String reason, Throwable error) {
-        String errorMsg = error != null && error.getMessage() != null ? error.getMessage() : reason;
-        return getPathError(errorMsg);
-    }
 
     /**
      * Returns error record for input reason and details. Error type is generic ballerina error type. This utility to
      * construct error struct from message.
```

Comment:
Change this to 
     * Returns error record for input reason and details.  This utility to construct error struct from the reason and message detail.